docs/packaging/updating-an-existing-package.md

```diff
@@ -43,3 +75,13 @@ After bumping or updating the package, build it:
 ```bash
 make
 ```
+
+## Building a package
+
+After bumping or updating the package, build it using the appropriate command. Usually this is:
+
+```bash
+make
+```
```

Comment:
Section above says pretty much the same thing.

Reply:
This was an oopsie from the rebase. I've removed the redundant text.